src/Microsoft.PowerShell.Commands.Utility/commands/utility/WebCmdlet/ConvertToJsonCommand.cs

```diff
@@ -100,7 +109,14 @@ protected override void BeginProcessing()
         /// </summary>
         protected override void ProcessRecord()
         {
-            _inputObjects.Add(InputObject);
+            if (ExperimentalFeature.IsEnabled("Microsoft.PowerShell.Utility.PSConvertToJsonRaw") && Raw && InputObject is PSObject)
+            {
+                _inputObjects.Add(((PSObject)InputObject).BaseObject);
```

Comment:
```suggestion
                _inputObjects.Add(obj.BaseObject);
```
this along with declaring `obj` above

Reply:
What if `obj` is a pure PSObject that is not backed up by a .NET object? I guess we should write out error in that case?